packages/parser/src/constants/bhaiLangSpec.ts

```diff
@@ -91,6 +97,11 @@ export const SPEC = [
   { regex: /^\bbas kar bhai\b/, tokenType: TokenTypes.BAS_KAR_BHAI },
   { regex: /^\bagla dekh bhai\b/, tokenType: TokenTypes.AGLA_DEKH_BHAI },
 
+  //functional programming
+  { regex: /^\bapna funda\b/, tokenType: TokenTypes.FUNDA_TYPE },
+  { regex: /^\brakh le bhai\b/, tokenType: TokenTypes.RAKH_LE_BHAI },
+  { regex: /^\w+(?=\(.*\))/, tokenType: TokenTypes.CALLABLE_TYPE },
```

Comment:
This regex doesn't considers any spaces after the function name and Open parenthesis. While spaces shouldn't bother function declaration. I would suggest to use the existing Identifier regex.

Spaces after the function name are valid in other languages also like python , js.

Also, there is no need to CALLABLE_TYPE token altogether.